.github/pull-request-template.md

```diff
@@ -1,4 +1,22 @@
-Fixes #(issue-number)
+<!-- PR tips
+
+New contributors:
+
+If you are new to Git/GitHub and want to make a quick fix to the docs,
+open your PR against the release branch where you found the error, such as
+"release-0.5".
+
+Experienced contributors:
+
+Most PRs should be opened against the master branch.
+
+If the change should also be in the most recent numbered release, add the
+corresponding "cherrypick-0.X" label; for example, "cherrypick-0.5." Best
```

Comment:
```suggestion
corresponding "cherrypick-0.X" label; for example, add "cherrypick-0.5" to PR-original. Best
```
Something to add clarity between all the references to PR?

Reply:
I added some wording to try to accomplish this.